lib/Receiver.js

```diff
@@ -345,18 +329,70 @@ class Receiver {
 
     return true;
   }
-}
 
-module.exports = Receiver;
+  /**
+   * Handles compressed data.
+   *
+   * @api private
+   */
 
-/**
- * Buffer utilities
- */
+  handleDataCompressed (packet) {
+    this.applyExtensions(packet, this.state.lastFragment, (err, buffer) => {
```

Comment:
Can we use `extension.decompress()` directly instead of wrapping it in `applyExtensions()`?